voxelsize:            0.1
noExtrusion:          False
intersections:        -1,                                                                                                            -1,        10000
pcm:                  "FLANN"
ransac:               False
decomposition:        "PMC"
optimizePlanes:       True
clusterPlanes:        False
cleanContours:        0
planeIterations:      3
fillHoles:            30
rda:                  0
pnt:                  0.85
smallRegionThreshold: 10
kd:                   50
ki:                   50
kn:                   50
mp:                   7
retesselate:          True
lft:                  0.01
generateTextures:     False
texMinClusterSize:    100
texMaxClusterSize:    0
textureAnalysis:      False
texelSize:            0.01
classifier:           "PlaneSimpsons"
recalcNormals:        False
threads:              8
sft:                  0.9
sct:                  0.7
reductionRatio:       0.0
tp:                   ""
co:                   ""
nsc:                  16
nccv:                 64
ct:                   50
colt:                 1.7976931348623157e+308
stat:                 1.7976931348623157e+308
feat:                 1.7976931348623157e+308
cro:                  False
patt:                 100
vcfp:                 False
useGPU:               False
flipx:                -1000000
flipy:                -1000000
flipz:                -1000000
